src/components/molecules/UserProfilePicture/UserReactions.scss

```diff
@@ -0,0 +1,115 @@
+@import "scss/constants";
+
+// TODO: should we move these to using CSS --variables rather than these SCSS vars
+$reactionSize: max(2.5vh, 2vw);
+$reactionOffset: calc(-1 * #{$reactionSize} / 2);
+$shoutBounceTranslateXOffset: 50%;
+
+.UserReactions {
+  // Note: we need this to ensure the reactions use this container for their positioning
+  position: relative;
+
+  width: fit-content;
+  height: fit-content;
+
+  &--reaction-left {
+    .UserReactions__reaction {
+      left: $reactionOffset;
+    }
+
+    .UserReactions__shout {
+      right: 50%;
+      transform-origin: right center;
+      animation: shout-bounce--left 5s ease;
+    }
+  }
+
+  &--reaction-right {
+    .UserReactions__reaction {
+      right: $reactionOffset;
+    }
+
+    .UserReactions__shout {
+      left: 50%;
+      transform-origin: left center;
+      animation: shout-bounce--right 5s ease;
+    }
+  }
+
+  &__reaction {
+    position: absolute;
+    top: $reactionOffset;
+
+    z-index: z(user-reactions-emoji);
+
+    // Note: emoji reactions are sized with font-size
+    font-size: $reactionSize;
+
+    animation: reaction-pulse 4s ease-in-out infinite;
+  }
+
+  &__shout {
+    position: absolute;
+    bottom: 0;
+
+    z-index: z(user-reactions-shout);
+
+    width: max-content;
+    max-width: 20em;
+    padding: 6px 10px;
+
+    background-color: rgba(255, 255, 255, 1);
```

Comment:
$white?

Reply:
I was originally keeping it this way to match how it was, but given it's 100% white, makes sense to just go with the basic `#fff` rather than `rgba`, so yeah, `$white` will work here.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/249f26615b9b3232440656652a9042c514d3ccea